build/tools/Automation/Commons/SdkStorageHelper.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
```

Comment:
nit: let's add a license header to this file

Reply:
Updated.